javaslang/src/test/java/javaslang/control/TryTest.java

```diff
@@ -211,21 +215,21 @@ public void shouldCreateFailureOnNonFatalException() {
 
     @Test
     public void shouldReturnAndNotThrowOnNonFatal() {
-        final Try.NonFatalException cause = Try.NonFatalException.of(new Exception());
-        assertThat(Try.NonFatalException.of(cause) instanceof Try.NonFatalException).isTrue();
+        final NonFatalException cause = NonFatalException.of(new Exception());
+        assertThat(NonFatalException.of(cause) instanceof NonFatalException).isTrue();
```

Comment:
\[FindBugs\] INFO: BC: instanceof will always return true for all nonnull values in javaslang\.control\.TryTest\.shouldReturnAndNotThrowOnNonFatal\(\), since all javaslang\.control\.Try$NonFatalException are instances of javaslang\.control\.Try$NonFatalException